CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Multiple QObjects can be defined in one bridge
-- Support for container types: `QSet<T>`
+- Support for container types: `QSet<T>`, `QHash<K, V>`
```

Comment:
need to add to the qml_features page too

and could show how to register custom combinations there too